classes/db/DbQuery.php

```diff
@@ -96,7 +96,7 @@ public function from($table, $alias = null)
             if ($table instanceof DbQuery) {
                 $query = '(' . $table->build() . ')';
             } else {
-                $query = '`' . _DB_PREFIX_ . $table . '`';
```

Comment:
This is this one that creates the UNIT test failure

Reply:
Maybe replace '`' by '\`' ?

Reply:
But I don't know why specifically in this case it causes a pb, I'll let you know if I find it

Reply:
The crash comes from the use of the Db Singleton in the pSQL function used itself by bqSQL().
The declaration of a Db class in the tests/Unit/Classes/RequestSqlTest.php file is not compliant with it. Why not considering a revision of this Test File ?